src/mumble/AudioInput.h

```diff
@@ -266,6 +266,7 @@ class AudioInput : public QThread {
 signals:
 	void doDeaf();
 	void doMute();
+	void doMuteCuePopup();
```

Comment:
Regarding the naming: I would use a name for this signal which does not imply the slot function that will be used. In the future we may want to do more things when the mute cue is triggered than just show the popup. So I would suggest a name as simple as ``doMuteCue``